src/libraries/System.IO.Compression.Brotli/src/System.IO.Compression.Brotli.csproj

```diff
@@ -4,10 +4,13 @@
     <OutputType>Library</OutputType>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <Nullable>enable</Nullable>
-    <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)-Unix</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)-Unix;$(NetCoreAppCurrent)-Browser</TargetFrameworks>
```

Comment:
Make this RID-less PNSE same as FileVersionInfo.

Reply:
I think in general, when we decide to add a PNSE because of lack of support for platform APIs we should do so by adding a RID-less PNSE.